guava/src/com/google/common/primitives/UnsignedInteger.java

```diff
@@ -195,7 +195,10 @@ public UnsignedInteger mod(UnsignedInteger val) {
 
   /** Returns the value of this {@code UnsignedInteger} as a {@code long}. */
   @Override
-  @SuppressWarnings("value:cast.unsafe") // Unknown long to PolyValue long is ok
+  @SuppressWarnings({
+    "value:cast.unsafe",
+    "cast.incomparable"
```

Comment:
Are there two warnings now? There should be two separate comments then. Also, is the incomparable warning for multiple type systems?

Also, if `toLong` returns the top qualifier Unknown, why is this no longer an unsafe downcast?